profitbricks-rest/src/main/java/org/apache/jclouds/profitbricks/rest/binder/BaseProfitBricksRequestBinder.java

```diff
@@ -16,20 +16,18 @@
  */
 package org.apache.jclouds.profitbricks.rest.binder;
 
-import static com.google.common.base.Preconditions.checkNotNull;
-
 import java.util.Map;
-
 import org.jclouds.http.HttpRequest;
 import org.jclouds.rest.MapBinder;
-
 import com.google.common.base.Strings;
-
 import org.jclouds.io.MutableContentMetadata;
 import org.jclouds.io.payloads.BaseMutableContentMetadata;
+import static com.google.common.base.Preconditions.checkNotNull;
 
 public abstract class BaseProfitBricksRequestBinder<T> implements MapBinder {
 
+   protected final String endpointPath = "/rest/v2";
```

Comment:
Oh, you can bind a value for test by adding a new module, like this:
```java
Injector injector = Guice.createInjector(new GsonModule(), new LocationModule(), new AbstractModule() {
   @Override protected void configure() {
      bind(String.class).annotatedWith(Names.named(Constants.PROPERTY_SESSION_INTERVAL)).toInstance("60");
   }
});
```
If it fails, could you paste the entire failure in a Gist?